<?php 
include_once '../conf/settings.inc.php';
include_once 'includes/header.inc.php'; 
include_once 'includes/quest_tutorial.inc';
?>

	<h3>EFI-EST Start Screen</h3>
<B>Note 1:</B>  The EFI uses the <A HREF="http://www.uniprot.org/">UniProtKB</A> protein
sequence database (maintained by <A HREF="http://www.ebi.ac.uk/">EMBL-EBI</A>)
for its annotations because, unlike <A HREF="http://www.ncbi.nlm.nih.gov/genbank/">GenBank</A>,
UniProtKB provides the ability to add to and/or correct functional
annotations.  In addition, the EFI uses the <A HREF="http://pfam.sanger.ac.uk/">Pfam</A>
and <A HREF="http://www.ebi.ac.uk/interpro/">InterPro</A> databases
(also maintained by EMBL-EBI) to provide easy access to the complete
memberships of a large number of curated protein
families/superfamilies (14,831 for Pfam; 25,586 for InterPro).  The
InterPro database collects signature sequences from 11 different
databases, including Pfam, to define its families.  Because the
different databases may define the “same” family with slightly
different signature sequences, InterPro families almost always are
larger than Pfam families.</P>
<br>
<P><b>Note 2:</B>  The sequence similarity networks generated by this webserver utilize the
full&#8209;length sequences of the proteins that are identified by
BLAST (Option A) or members of specified Pfam and/or InterPro
families (Option B). As a result, the clusters that are generated and
visualized in the networks will result from sequence similarities for
the entire sequence. However, many proteins have multiple domains;
for these proteins the alignments used to calculate the alignment scores will
not necessarily be for the domain in which you may be interested. The
EFI plans to provide the capability to excise structure-based domains
as defined by CATH/SCOP from multidomain proteins; however, this has
not yet been incorporated. So, you must be aware of this limitation
when interpreting the networks provided by EFI-EST.</P>
<br>
<p>In addition to the tutorial link at the top of the screen, the start
screen contains two input fields for generating the data set from
which your networks will be created. </P>

<br>
<p><B>Option A: Networks for a “local” dataset.</B>  Paste a protein sequence (without a FASTA header) into the input
box. A sequence data set will be built containing the most closely
related sequences retrieved from the <A HREF="http://www.uniprot.org/">UniProtKB</A>
database using a BLAST e&#8209;value upper limit threshold of 10<sup>-5</sup>.
A maximum of <?php echo number_format(__MAX_BLAST_SEQ__,0); ?> sequences will be used, but the data set may be
smaller if &lt;<?php echo number_format(__MAX_BLAST_SEQ__,0); ?> sequences are found using a BLAST alignment score upper
limit of 10<sup>-5</sup>.
 A maximum of <?php echo number_format(__MAX_BLAST_SEQ__,0); ?> sequences is used because in most cases a full
network with all sequences (nodes) will be viewable without having to
collapse nodes into representative nodes (explained <a href='tutorial_download.php'>here</a>). Use this option if you are only interested in those
proteins that are most similar to your protein of interest.</P>
<p><br><img src='images/Start_Screen_Figure1.jpg' width='700'><br></p>
<p></p>
<P><b>Option B:  Networks for an entire (super)family.</B>
Enter the <A HREF="http://pfam.sanger.ac.uk/">Pfam</A> and/or
<A HREF="http://www.ebi.ac.uk/interpro">InterPro</A> family numbers
in the input box for Option B in a comma separated list.  If you do
not know the Pfam and/or InterPro, you can find them by entering a
sequence in the “Analyse your protein sequence” field on the
<A HREF="http://www.ebi.ac.uk/interpro/">InterPro homepage</A>.</P>

<p><br><img src='images/Start_Screen_Figure2.jpg' width='700'><br></p>
<p>The signature sequences in the InterPro database will be searched; the
output is a graphic showing the matches to the signature sequences in
the 11 databases (and to what regions of the protein the match
occurs). </P>
<p><br><img src='images/Start_Screen_Figure3.jpg' width='700'></br></p>

<P>The family ID is given for each match; you can click on the database IDs to access the
detailed description of the family in that database. </P>

<p><br><img src='images/Start_Screen_Figure4.jpg' width='700'><br></p>
<P>At present, the EFI-EST server accepts only Pfam and InterPro families.
For Pfam families, the format is PFxxxxx (five digits); for InterPro
families, the format is IPRxxxxxx (six digits).  Option B usually
will result in a much larger data set because all of the members of
(super)families are included.  Because entire (super)families are in
the dataset, it is likely that you'll need to view representative
node SSNs instead of full SSNs (see below).</p>

<P>The number of sequences that can be used in Option B is limited to
<?php echo number_format(__MAX_SEQ__,0); ?>. This limit is set to ensure that assembling the
dataset/performing the all-by&#8209;all BLAST as well as generating
the networks can be completed within several hours. </P>

<P>When input has been entered for either Option, enter your e-mail address
(for data retrieval only), and hit “Go” at the bottom of the
screen. EFI-EST will begin assembling the dataset and performing the
all-by&#8209;all BLAST. The all-by-all BLAST calculation will return
alignment scores/edges for those sequence pairs for which the BLAST e-values are
less than an upper limit threshold of 10<sup>-5</sup>.  For most (super)families, this threshold should provide sufficient
internode connections (edges) in the networks that inferences about
divergent evolution of protein function are possible.</P>

<p><br><img src='images/Start_Screen_Figure5.jpg' width='700'><br></p>
<P>Since the all-by-all BLAST may take several hours (depending on the size
and sequence divergence of the dataset), you may close the running
window. When the dataset is complete, you’ll receive an e-mail with
a link to analyze the dataset. This link will be active for 7 days so
that you may return at your convenience.</P>

<P><B>Note:</B>  If you are interested in exploring sequence-function relationships in
(super)families with &gt <?php echo number_format(__MAX_SEQ__,0); ?> sequences, please send an
e-mail to efi@enzymefunction.org with a brief summary of your
interests. We will provide access to the EFI’s Unix scripts for
generating networks on Biocluster at the Institute for Genomic
Biology at the University of Illinois, Urbana&#8209;Champaign as well
as assistance in using the scripts.</P>


        <p>&nbsp;</p>
<div>
  <div></div>
</div>
	<form action='tutorial_analysis.php' method='post'>
        <button type='submit' class='css_btn_class'>CONTINUE WITH THE TUTORIAL</button>
</form>

    <?php require('includes/gobackto_quest.php'); ?>

    </div>

<?php include_once 'includes/footer.inc.php'; ?>

